index.html

```diff
@@ -3187,7 +3187,15 @@ <h3 id="h-sourcebufferlist-events" resource="#h-sourcebufferlist-events"><span p
         </thead>
         <tbody>
           <tr>
-            <td>06 June 2016</td>
+            <td>07 June 2016</td>
```

Comment:
I thought the "Squash and Merge" function removed the need for doing a separate rebase, and that all we needed to do was resolve merge conflicts before initiating the merge.  Should we also run "git rebase gh-pages" on the branch before we merge?